### PR TITLE
Remove roles from @Authenticated

### PR DESCRIPTION
**Pull request by ryanhall07**
All callers have been updated to use capabilities instead

### CODE REVIEW

File: misk/src/main/kotlin/misk/web/actions/WebActionMetadataAction.kt [hunk 1]
```diff
@@ -49,7 +49,6 @@ data class WebActionMetadata(
   val networkInterceptors: List<String>,
   val dispatchMechanism: DispatchMechanism,
   val allowedServices: Set<String>,
-  val allowedRoles: Set<String>,
```
Comment (ryanhall07) [inline misk/src/main/kotlin/misk/web/actions/WebActionMetadataAction.kt]: @adrw is this ok to remove? are other repos reading this field?

File: misk/web/tabs/web-actions/src/containers/WebActionCardContainer.tsx [hunk 2]
```diff
@@ -68,7 +68,7 @@ const WebActionCardContainer = (
               tag={`${props.tag}::Services`}
             />
             <MetadataCollapse
-              content={props.action.allowedRoles}
+              content={props.action.allowedCapabilities}
```
Comment (ryanhall07) [inline misk/web/tabs/web-actions/src/containers/WebActionCardContainer.tsx]: @adrw can you double check all the web stuff. I basically just did a sed of roles -> capabilities and science dog'd it.

**Comment by ryanhall07** [comment]
Closes https://github.com/cashapp/misk/issues/1078

File: misk/web/tabs/web-actions/src/containers/WebActionCardContainer.tsx (hunk 2 above)
Comment (adrw) [reply misk/web/tabs/web-actions/src/containers/WebActionCardContainer.tsx]: Looks good from my initial testing and code review 👍

**Pull request merged**